packages/realm-server/node-realm.ts

```diff
@@ -145,6 +147,22 @@ export class NodeAdapter implements RealmAdapter {
     onClose(request, cleanup);
     return { response, writable: s as unknown as WritableStream };
   }
+
+  createJWT(claims: TokenClaims, expiration: string, secret: string): string {
+    let token = jwt.sign(claims, secret, { expiresIn: expiration });
+    return token;
+  }
+
+  verifyJWT(
+    token: string,
+    secret: string,
+  ): TokenClaims & { iat: number; exp: number } {
+    // throws TokenExpiredError and JsonWebTokenError
+    return jwt.verify(token, secret) as TokenClaims & {
+      iat: number;
+      exp: number;
+    };
+  }
```

Comment:
@jurgenwerk i recall you mentioned picking up the work to verify the token on the server. this is the realm adapter method that you'll want to use to verify the JWT that the client sends.

Reply:
Got it!